Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1101,24 +1114,6 @@ async def inner():
         with self.assertRaises(FooException):
             loop.run_until_complete(foo())
 
-    def test_wait_for_raises_timeout_error_if_returned_during_cancellation(self):
```

Comment:
But, then this test literally tests that we don't return a value when it has completed successfully...

I can't see any reason for this test, the issue linked to the PR does not mention anything that warrants this test existing:
https://bugs.python.org/issue40607

Reply:
So, as far as I can tell, this test should be binned and the behaviour should be as defined in the above changes.